modules/ROOT/pages/notifications/index.adoc

```diff
@@ -57,11 +57,11 @@ The following categories exist, ordered by severity:
 | SECURITY
 | WARNING OR INFORMATION
 | The result of the query or command indicates a potential security issue.
-| Make sure that the behaviour is what you intended.
+| Make sure that the behavior is what you intended.
 
 | TOPOLOGY
 | INFORMATION
-| Information provided while executing database and server related commands.
+| Information provided while executing database and server-related commands.
```

Comment:
Do you think we should update the GENERIC one to also not say 'wider class' but 'another category' or something similar, or do you think it is ok to use the word 'class' there? 

Also a side note, drivers team is at least partly using these descriptions in their configuration API, not sure if they like to adapt to language changes we do here.

Reply:
I wasn't sure if this was something from the code. I can remove the dash.